flowmachine/tests/functional_tests/conftest.py

```diff
@@ -1,31 +1,9 @@
-import pandas as pd
 import pytest
 
 from approvaltests.reporters.generic_diff_reporter_factory import (
     GenericDiffReporterFactory,
 )
 
-import flowmachine
-from flowmachine.core import Query
-from flowmachine.core.cache import reset_cache
-
-
-@pytest.fixture(autouse=True)
-def flowmachine_connect():
-    con = flowmachine.connect()
-    yield con
-    reset_cache(con)
-    con.engine.dispose()  # Close the connection
-    Query.redis.flushdb()  # Empty the redis
-    del Query.connection  # Ensure we recreate everything at next use
-
-
-@pytest.fixture
-def get_dataframe(flowmachine_connect):
```

Comment:
I assume these can be removed because they already exist in `conftest.py` in the parent directory? Good catch!